R/udf_api.R

```diff
@@ -858,7 +858,7 @@ UdfApi <- R6::R6Class(
       }
 
       urlPath <- "/udfs/generic/{namespace}"
-      if (!missing(`namespace`)) {
+      if (!is.null(`namespace`)) {
```

Comment:
This is a little bit that should have gone in with https://github.com/TileDB-Inc/TileDB-Cloud-R/pull/104 just now